environment_linux.yml

```diff
@@ -21,7 +21,7 @@ dependencies:
   - dask # same as dask[complete] pip distribution
   - python-graphviz
   # core functionality and APIs
-  - matplotlib>=3.3.3
+  - matplotlib==3.4.2
```

Comment:
matplotlib API has changed, so we have to force the version now, to latest, for consistent behavior